frontend/www/js/omegaup/components/user/PasswordEdit.vue

```diff
@@ -65,16 +69,47 @@ import T from '../../lang';
 export default class UserPasswordEdit extends Vue {
   T = T;
   oldPassword = '';
-  newPassword1 = '';
+  newPassword = '';
   newPassword2 = '';
+  passwordMismatch = false;
+
+  get invalidPasswordClass(): string {
+    return this.passwordMismatch ? 'invalid-input' : '';
+  }
 
   onUpdatePassword(): void {
-    this.$emit(
-      'update-password',
-      this.oldPassword,
-      this.newPassword1,
-      this.newPassword2,
-    );
+    if (this.newPassword !== this.newPassword2) {
+      this.passwordMismatch = true;
+      return;
+    }
```

Comment:
lo que debería pasar es que cuando cualquiera de `newPassword` o `newPassword2` se cambien, `this.passwordMismatch` se debería actualizar. luego aquí esto debería ser

```suggestion
    if (this.passwordMismatch) {
      return;
    }
```

también hay que deshabilitar el botón cuando `this.passwordMismatch` sea verdadero o la contraseña sea vacía.